Resources/Private/Translations/de/Main.xlf

```diff
@@ -762,6 +861,184 @@
                 <target>Gehe zu nächster Seite</target>
             </trans-unit>
 
+            <!-- error handling -->
+
+            <trans-unit id="errors.internal.title" xml:space="preserve" approved="yes">
+                <source>Internal server error</source>
+                <target>Interner Serverfehler</target>
+            </trans-unit>
+            <trans-unit id="errors.graphql.title" xml:space="preserve" approved="yes">
+                <source>Communication error</source>
+                <target>Kommunikationsfehler</target>
+            </trans-unit>
+            <trans-unit id="errors.1603921233w.title" xml:space="preserve" approved="yes">
+                <source>Failed to create tag</source>
+                <target>Tag-Erstellung fehlgeschlagen</target>
+            </trans-unit>
+            <trans-unit id="errors.1603921233.message" xml:space="preserve" approved="yes">
+                <source>This tag is already exists. Please choose a different one.</source>
+                <target>Dieser Tag existiert bereits. Bitte wählen Sie einen anderen aus.</target>
+            </trans-unit>
+            <trans-unit id="errors.1509632861.message" xml:space="preserve" approved="yes">
+                <source>The specified asset was not found.</source>
+                <target>Das angegebene Asset wurde nicht gefunden.</target>
+            </trans-unit>
+            <trans-unit id="errors.1678330583.message" xml:space="preserve" approved="yes">
+                <source>Parent must be an AssetCollection</source>
+                <target>Übergeordnetes Element muss eine Asset-Sammlung sein</target>
+            </trans-unit>
+            <trans-unit id="errors.1678085489.message" xml:space="preserve" approved="yes">
+                <source>Invalid metadata definition</source>
+                <target>Ungültige Metadaten-Definition</target>
+            </trans-unit>
+            <trans-unit id="errors.1591537315.message" xml:space="preserve" approved="yes">
+                <source>Failed to delete asset</source>
+                <target>Asset-Löschung fehlgeschlagen</target>
+            </trans-unit>
+            <trans-unit id="errors.1590659044.message" xml:space="preserve" approved="yes">
+                <source>Cannot update asset that was never imported</source>
+                <target>Asset kann nicht aktualisiert werden, da es nie importiert wurde</target>
```

Comment:
Please adjust `Asset` to `Datei` in German in the remaining file, and `collection` to `Sammlung`.